.github/workflows/deploy-pages.yml

```diff
@@ -0,0 +1,55 @@
+name: deploy-pages
+on:
+  push:
+    branches:
+      - develop
+  workflow_dispatch:
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: pages
+  cancel-in-progress: false
+
+jobs:
+  build-docs:
+    uses: ./.github/workflows/doc.yml
+  build-site:
+    runs-on: ubuntu-latest
+    needs: build-docs
+    steps:
+      - name: Check out Celeritas doc base
+        uses: actions/checkout@v4
+        with:
+          ref: doc/gh-pages-base
```

Comment:
Do we have a note of this branch existence in a onboarding doc for (new) developers?

Reply:
I've added a note about it to the administration section. Basically no one should need to know about or touch this branch, since it is only there to redirect to the dev/user docs. Someday we could update it to include release notes, people description, etc.

Reply:
> I've added a note about it to the administration section.

Thanks.

> Basically no one should need to know about or touch this branch

Which also means the project will eventually forgot about it existence until something breaks :)  (But this is fixes by adding to the administration section).